etc/optimus.yaml

```diff
@@ -49,6 +49,12 @@ workers:
     # Optional.
     # Default value is false.
     # dry_run: true
+    # Identity level specifies which kind of customers this bot allows to work
+    # with.
+    # Required.
+    # Possible values are: anonymous, registered, identified and
+    # professional.
```

Comment:
just curious - what's the purpose of newline here?

Reply:
80 margin limit